README.md

```diff
@@ -59,14 +113,14 @@ The library implements a constant time comparison algorithm for preventing timin
 To enable `FastSecureCompare` in your environment, add the following to your Gemfile:
```

Comment:
As currently formatted, this `Performance` section is specific to the Android Pay instructions. I believe it applies to both approaches? If so, just change `### Performance` to `## Performance` 👍

Reply:
DOH 
👍